src/worker.js

```diff
@@ -2,31 +2,51 @@ const { workerData, parentPort } = require("worker_threads");
 const chalk = require("chalk");
 const fs = require("fs").promises;
 const { ESLint } = require("eslint");
-
 const eslint = new ESLint(workerData.options);
-
+const path = require("path");
 const formatterPromise = workerData.formatter
   ? eslint.loadFormatter(workerData.formatter)
   : undefined;
 
+function getPrettyPath(absolutePath) {
+  let relativePath = path.relative(process.cwd(), absolutePath);
+  let containingDir = path.dirname(relativePath);
+
+  if (containingDir && containingDir !== '.') {
+      relativePath = path.join(path.basename(containingDir), path.basename(relativePath));
+  }
+
+  return relativePath;
+}
+
+function logMessage(message) {
+  console.log(
+    chalk.dim(new Date().toLocaleTimeString()),
+    chalk.bold(chalk.cyan("[vite-plugin-eslint]")),
```

Comment:
I'm unsure about this long prefix in the console for something that happens quite often.